sdk/sqlmanagement/Azure.ResourceManager.Sql/src/autorest.md

```diff
@@ -4,7 +4,7 @@ Run `dotnet build /t:GenerateCode` to generate code.
 
 ``` yaml
 azure-arm: true
-require: https://github.com/Azure/azure-rest-api-specs/blob/55090ea4342b5dac48bc2e9706e3a59465ffa34c/specification/sql/resource-manager/readme.md
+require: https://github.com/Azure/azure-rest-api-specs/blob/7b68a8cd8cb3f2328d4830fbbac41d8ad5febb67/specification/sql/resource-manager/readme.md
 tag: package-composite-v5
```

Comment:
Not sdk issue, but question about the swagger, why an update can apply on an existing tag but not a new tag

Reply:
hmmm.... we update SDK tag every 12 months~ 
v5 sdk tag has been created since May 2021. 
v6 SDK tag will be created after we release the new stable version: 2021-11-01-stable (our stable version is 6-9 months behind previe version)